src/ecalc/libraries/libecalc/fixtures/libecalc/fixtures/cases/consumer_system_v2/consumer_system_v2_dto.py

```diff
@@ -16,7 +30,7 @@ def consumer_system_v2_dto() -> DTOCase:
         datetime(2022, 1, 1, 0, 0): dto.FuelType(
             name="fuel_gas",
             price=Expression.setup_from_expression(1.5),
-            user_defined_category="FUEL-GAS",
+            user_defined_category=FuelTypeUserDefinedCategoryType.FUEL_GAS,
```

Comment:
ok to use the enum? should not be wrapped in `str(<enum>)`? the constructor specifies enum or str?

Reply:
The type is the Enum. String should not be valid. It works because of pydantic magic